src/main/java/com/google/cloud/tools/io/FilePermissions.java

```diff
@@ -41,6 +42,7 @@
   public static void verifyDirectoryCreatable(Path path)
       throws AccessDeniedException, NotDirectoryException {
 
+    Preconditions.checkNotNull(path, "Null directory path");
```

Comment:
Could add `@throws NPE` to the Javadoc.

Reply:
We normally assume this by default and only document it if the argument is Nullable.